src/java/org/apache/cassandra/batchlog/BatchlogManager.java

```diff
@@ -457,6 +458,7 @@ private void writeHintsForUndeliveredEndpoints(int startFrom, Set<InetAddressAnd
             Keyspace keyspace = Keyspace.open(ks);
             Token tk = mutation.key().getToken();
 
+            // TODO: this should be done through adding "non-avlie" replicas to replica layout
```

Comment:
Adds a TODO and a spelling error can we resolve this?

Reply:
It's not been resolved, but there's no need to keep this TODO there specifically - it seems to document the follow-ups to ReplicaLayout Alex and I discussed, that I would very much like to see make it to the codebase, but will have to happen post-4.0